src/main/java/com/snowflake/kafka/connector/records/SnowflakeAvroConverterWithStringFallback.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019 Snowflake Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.snowflake.kafka.connector.records;
+
+import com.snowflake.kafka.connector.internal.Logging;
+import org.apache.kafka.connect.data.SchemaAndValue;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public class SnowflakeAvroConverterWithStringFallback extends SnowflakeAvroConverter
+{
+  @Override
+  SchemaAndValue handleError(final Exception e, final byte[] bytes)
+  {
+    LOGGER.debug(Logging.logMessage("failed to parse record as Avro, interpreting as utf-8 string"));
+    // assumes byte array is UTF-8 encoded
+    String wrappedJson = "{\"rawBytes\": \"" + new String(bytes, StandardCharsets.UTF_8) + "\"}";
```

Comment:
I think it's necessary to escaping string inside JSON. For example, if `bytes` will contains `"` then there will be invalid json in Snowflake.

Reply:
I guess we expect to break if there will be invalid JSON.

Reply:
thanks good catch! Updated to have jackson perform the string escape via `mapper.valueToTree(payload)` call

Reply:
Thank you!